pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1002,7 +1005,7 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 			if len(parseVolumeTypeInfo) >= 2 {
 				newReplicaCount, err := convertVolumeParam(parseVolumeTypeInfo[1])
 				if err != nil {
-					return nil, fmt.Errorf("error [%v] when parsing value %q of option '%s' for volume plugin %s", err, parseVolumeTypeInfo[1], "volumetype", glusterfsPluginName)
+					return nil, fmt.Errorf("error %v when parsing value %q of option '%s' for volume plugin %s", err, parseVolumeTypeInfo[1], "volumetype", glusterfsPluginName)
```

Comment:
`"error parsing volumeType %q: %s", parseVolumeTypeInfo[1], err"

IMO, the plugin name is not relevant.

(applies to all volumeType errors below)

Reply:
Done.